packages/ui/src/components/Select/select.tsx

```diff
@@ -64,17 +77,33 @@ const Select: ISelect = (props) => {
         <DefaultSelect
             {...rest}
             id={id}
+            mode={isMultiple ? 'multiple' : undefined}
             prefixCls={SELECT_CLASS_PREFIX}
             className={className}
             suffixIcon={<ChevronDown size='small' />}
             onChange={handleChange}
+            removeIcon={<Close size='small' />}
+            clearIcon={<XCircle size='small' />}
+            menuItemSelectedIcon={<Check size='small' />}
+            showArrow
+            notFoundContent={
+                <>
+                    <Inbox size='large' />
+                    {notFoundContentLabel && <Typography.Text size='medium'>
+                        {notFoundContentLabel}
+                    </Typography.Text>}
+                </>
+            }
         >
             {options
-                ? options.map(({ label, value, textType, disabled }, optionIndex) => (
-                    <Option key={optionIndex} value={value} disabled={disabled}>
-                        <Typography.Text size='medium' disabled={disabled} type={textType}>
-                            {label}
-                        </Typography.Text>
+                ? options.map(({ label, value, textType, disabled, title }, optionIndex) => (
+                    <Option
+                        key={optionIndex}
+                        value={value}
+                        disabled={disabled}
+                        title={(title || typeof label !== 'string') ? title : label}
```

Comment:
```
title={title || !isString(label) ? title : label}
```

Reply:
`... || ... ? ... : ...` without grouping condition seems a bit more dirty to me